python/sanlock.c

```diff
@@ -1686,7 +1686,7 @@ __module_init(PyObject* m)
     }
 
     Py_INCREF(py_exception);
-    if (PyModule_AddObject(m, "SanlockException", py_exception)) {
+    if (PyModule_AddObject(m, "sanlockException", py_exception)) {
```

Comment:
This is an API change that will break most users of sanlock. We can never change the exception name.

Think about all the code in vdsm doing:

    try:
        ...
    except sanlock.SanlockExcpetion:
        ...

Will fail with NameError when sanlock.SanlockException is raised, hiding the original error from sanlock.

Reply:
will revert